specifyweb/specify/views.py

```diff
@@ -5,29 +5,25 @@
 import json
 import mimetypes
 from functools import wraps
-from itertools import groupby
-from typing import Any, Callable, Dict, List, Optional, Union
+from typing import Any, Callable, Dict, List, Union
 from uuid import uuid4
 import traceback
```

Comment:
(optional)
The `traceback` module is now never used in this file.
Could this import be removed?